tools/pythonpkg/duckdb_python.cpp

```diff
@@ -1189,6 +1197,10 @@ struct DuckDBPyRelation {
 };
 
 PYBIND11_MODULE(duckdb, m) {
+	m.attr("apilevel") = "1.0";
```

Comment:
any reason those attributes were added?

Reply:
Yeah, they're part of PEP 249 for DBAPI 2.0 (https://www.python.org/dev/peps/pep-0249/#globals ), but they didn't need to be in this PR.

Reply:
Ah thanks! No worries I was just wondering.